### DISCUSSION TITLE
Laravel 11

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#881] [General] [answered]
What about Backpack & Laravel 11 ?
:)

**Comment by pxpm** [comment, marked as answer]
@blondie63 working on it 👍 

Hope to have it released by the end of the week. 

Cheers

**Comment by jcastroa87** [comment]
Soon, soon! I will close the issue because the question is answered, but next week you will be able to install Laravel 11 + Backpack. Please feel free to re-open or create a new one if needed.

Cheers.
